ykman/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor: Daniel Everett <deverett@gmail.com>
+# Maintainer: Daniel Everett <deverett@gmail.com>
+pkgname=yubikey-manager
+pkgver=0.7.1
+pkgrel=0
+pkgdesc="Python library and command line tool for configuring any YubiKey over all USB interfaces"
+url="https://github.com/dae3/alpine-yubikey-manager"
```

Comment:
`url` should be homepage of the packaged software, that's https://developers.yubico.com/yubikey-manager/. You're repo with APKBUILD is not upstream.